2Homework25.02.22/Trie/Trie/Trie.cs

```diff
@@ -0,0 +1,148 @@
+namespace Trie;
+
+/// <summary>
+/// data structure for storing a set of strings
+/// </summary>
+public class Trie
+    {
+        private class Node
+        {
+            public Node()
+            {
+                this.Next = new Dictionary<char, Node>();
+            }
+            public short NumberOfWords;
+            public bool IsTerminal;
+            public Dictionary<char, Node> Next;
+        }
+        public Trie()
```

Comment:
```suggestion
        }
        
        public Trie()
```
Все члены класса пустой строкой разделяются